commit caa5a6bed31937f2d1b322da204e11eae57a720f
Author: Nicolas Berbiche <nicolas@normie.dev>
Date:   Tue Oct 20 18:14:44 2020 -0400

    PATCH: use nixpkgs utf8proc
    
    This patch forces the Makefile to use utf8proc from `buildInputs`.
    The Makefile relies on ldconfig to find the utf8proc libraries,
    which is not possible with nixpkgs.

diff --git a/Makefile b/Makefile
index caccdf7..90b11ea 100644
--- a/Makefile
+++ b/Makefile
@@ -29,7 +29,7 @@ else
 	LDFLAGS = -O2 -flto
 endif
 
-ifeq ($(shell ldconfig -p | grep libutf8proc.so > /dev/null || echo fail),fail)
+ifeq (false,fail)
 $(info libutf8proc not found. Support for language-specific combining characters and unicode normalization will be disabled.)
 	CFLAGS += -DNOUTF8PROC
 else

